runners/apex/pom.xml

```diff
@@ -187,6 +187,7 @@
               <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
               <excludedGroups>
                 org.apache.beam.sdk.testing.UsesStatefulParDo,
+                org.apache.beam.sdk.testing.UsesTimersInParDo,
```

Comment:
(Gonna merge to move on, since the rename is probably bigger, or of the stuff _not_ touched here, or both)